shippable.yml

```diff
@@ -12,3 +15,11 @@ build:
    - shippable_retry pip install -Ur test-requirements.txt
 
    - pytest
+
+after_success:
+
+  - codecov --token=272fc7a8-bd55-4e0c-af2f-7d475d0f3c0a -r ansible/ansibullbot
+
+  - coverage xml -o $COVERAGE_REPORTS/coverage.xml
```

Comment:
What are these supposed to do? pytest already puts a report in that folder. See `pytest.ini`.